components/card.html

```diff
@@ -0,0 +1,207 @@
+                        style="width: 1024px; height: 1050px; border:0; transform: scale(1); overflow:hidden;"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                <hr style="width:50%;text-align:center">
+
+            <h1 class="main-heading-center">Icon Button</h1>
+            <div class="component-position main-heading-center">
+            
+                <div class="container1">
+                    
+                    <img src="/img/img_avatar.png" width=200px alt="image" />
+                    <h3>Product Name</h3>
+                    <div class="prod-details">
+                      
+                         MRP:<div class="line-through"><i class="fas fa-rupee-sign"></i>
+                      1000</div>&ensp;300
```

Comment:
&ensp?

Reply:
&ensp was used for spacing , actually it is equal to approx. 4 spaces

Reply:
okay